src/Pact/Types/Gas.hs

```diff
@@ -44,6 +44,7 @@ data GasArgs
   | GReduced [Term Name]
   | GUse ModuleName (Maybe Hash)
   | GModule Module
+  | GInterface Module
```

Comment:
Note for future: we need a safe ADT for `Module` so we can stop excessive casing

Reply:
Agreed - that would be very nice.